test/testmemleak.cpp

```diff
@@ -2242,6 +2242,35 @@ class TestMemleakStructMember : public TestFixture {
             "    return 0;\n"
             "}", true);
         ASSERT_EQUALS("", errout.str());
+
+        check(
```

Comment:
I don't know where you should put these test cases but it does not seem that `lambdaInForLoop` is the proper place.

Reply:
Well, my excuse is that this is the test case for [#9793](https://trac.cppcheck.net/ticket/9793), which caused the FN.

Reply:
ok then it definitely makes sense to put the test cases together. the function name is misleading so feel free to rename it somehow if you can think of something..